docs/en_US/ResearchBlog/NASComparison.md

```diff
@@ -0,0 +1,81 @@
+
+
+
+| Dataset                                                      | Training Size | Numer of Classes | Descriptions                                                 |
+| :----------------------------------------------------------- | ------------- | ---------------- | ------------------------------------------------------------ |
+| [Fashion-MNIST](<https://github.com/zalandoresearch/fashion-mnist>) | 60,000        | 10               | T-shirt/top, trouser, pullover, dress, coat, sandal, shirt, sneaker, bag and ankle boot. |
+| [CIFAR-10](<https://www.cs.toronto.edu/~kriz/cifar.html>)    | 50,000        | 10               | Airplanes, cars, birds, cats, deer, dogs, frogs, horses, ships and trucks. |
+| [CIFAR-100](<https://www.cs.toronto.edu/~kriz/cifar.html>)   | 50,000        | 100              | Similar to CIFAR-10 but with 100 classes and 600 images each. |
+| [OUI-Adience-Age](<https://talhassner.github.io/home/projects/Adience/Adience-data.html>) | 26,580        | 8                | 8 age groups/labels (0-2, 4-6, 8-13, 15-20, 25-32, 38-43, 48-53, 60-). |
+| [ImageNet-10-1](<http://www.image-net.org/>)                 | 9,750         | 10               | Coffee mug, computer keyboard, dining table, wardrobe, lawn mower, microphone, swing, sewing machine, odometer and gas pump. |
+| [ImageNet-10-2](<http://www.image-net.org/>)                 | 9,750         | 10               | Drum, banj, whistle, grand piano, violin, organ, acoustic guitar, trombone, flute and sax. |
+
+
+
+We do not change the default fine-tuning technique in their source code. In order to match each task, the codes of input image shape and output numbers are changed.  
+
+Search phase time for all NAS methods is **two days** as well as the retrain time.  Average results are reported based on **three repeat times**.
+
+For NAO, it requires too much computing resources, so we only use NAO-WS which provides the pipeline script.
+
+For AutoKeras, we used  0.2.18 version because it was the latest version when we started the experiment.
+
+## NAS Performance 
+
+| NAS             | AutoKeras (%) | ENAS (macro) (%) | ENAS (micro) (%) | DARTS (%) | NAO-WS (%) |
+| --------------- | :-----------: | :--------------: | :--------------: | :-------: | :--------: |
+| Fashion-MNIST   |     91.84     |      95.44       |      95.53       | **95.74** |   95.20    |
+| CIFAR-10        |     75.78     |      95.68       |    **96.16**     |   94.23   |   95.64    |
+| CIFAR-100       |     43.61     |      78.13       |      78.84       | **79.74** |   75.75    |
+| OUI-Adience-Age |     63.20     |    **80.34**     |      78.55       |   76.83   |   72.96    |
+| ImageNet-10-1   |     61.80     |      77.07       |      79.80       | **80.48** |   77.20    |
+| ImageNet-10-2   |     37.20     |      58.13       |      56.47       |   60.53   | **61.20**  |
+
+
+
+Unfortunately, we cannot reproduce all the results in the paper. 
```

Comment:
is this only happen to us? how big the gaps?

Reply:
I believe it happens to every users who want to reproduce the results. The gaps of accuracy are 12.78%, 0.45%,  0.95%, 2.94% and 0.83%  for AutoKeras, ENAS(macro), ENAS(micro), DARTS and NAO-WS. It‘ not strange here because I get an average result from three runs while it usually reports the best accuracy in the paper.